gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -915,21 +915,23 @@ service Gateway {
 
   /*
     Migrates the process instance to the specified process definition.
-    In simple terms, this is handled by updating the active element's process and scope keys.
+    In simple terms, this is handled by updating the active element's process.
 
     Errors:
       NOT_FOUND:
-        -the process instance referenced by `processInstanceKey` must be active in Zeebe
+        - the process instance referenced by `processInstanceKey` is not active
 
       FAILED_PRECONDITION:
-        - all active elements in the source process instance should be mapped to the elements in the target process
-        - mapping instructions cannot change the element type (or event type)
-        - mapping instructions must refer to supported elements (i.e. some elements will be supported later on)
-        - mapping instructions must refer to elements in supported scenarios.
+        - not all active elements in the source process instance are mapped to the elements in the target process
+        - mapping instructions doesn't map the same type of elements or events
+        - mapping instructions doesn't refer to supported elements (i.e. some elements will be supported later on)
+        - mapping instructions doesn't refer to elements in supported scenarios.
```

Comment:
🔧 Typically it's easier to understand statements when they are in a positive form rather than a negative.

E.g. `a mapping instruction changes the type of an element or event` can be considered easier to understand than `a mapping instruction doesn't map the same type of elements or events`

Reply:
That is actually true. Updated! 👍